### ISSUE TITLE
Mini Golfe Penha | Guimarães

### ISSUE THREAD

**Issue by nelsonic** [bmhco/shared#6]
This is not strictly related to BMH Co. 
Rather it's a weekend activity to do with our 👶 that he will probably _love_! 😍 
Opening it here as a reminder...

As a child, Mini Golf was one of my things to do:

![image](https://user-images.githubusercontent.com/194400/205930448-5a8f0b40-f9af-467a-b044-2b76f150c2ed.png)

> **Note**: not specifically, **`this`** mini golf location, I didn't grow up in the north of PT ... but in general.

https://www.minigolfe.pt/campos-de-minigolfe/braga/minigolfe-da-penha/
<img width="1254" alt="image" src="https://user-images.githubusercontent.com/194400/205924233-dc3e3bba-2dbf-4104-92ad-0c7d8d595efe.png">

Yes, perhaps unsurprisingly, there is a website dedicated to this wonderful sport. 😜 

Sadly no website. just this [Facebook](https://github.com/dwyl/learn-react/issues/23#issuecomment-925783203) page: https://www.facebook.com/penhaminigolfe 

This slideshow video is the best I've found to give you an idea of the course: https://youtu.be/30omGo1vjK8
Scrub through it, don't watch it at "normal speed" 🐌 
The holes appear to be earth not concrete or astroturf. 🤷‍♂️ (_what happens when it rains ..._ 🌧️)

# Horário:

Horário de verão: Terça-feira a Domingo, das 11h às 20h.
Horário de inverno – Sábado, Domingo e Feriados, das 11h às 19h

So basically it's an afternoon activity. 

# Contact:

**`253 064 547`**
**`963 069 726`**

# Location

Rua da Floresta
4810-038 Guimarães

https://goo.gl/maps/9sCvMVQoLTu1zVTo8 **`T30mins`** each way. 
<img width="1205" alt="image" src="https://user-images.githubusercontent.com/194400/205925019-43d243fd-7d5e-4b56-868f-104bf4222831.png">

We could go there on a Sunday after nap time.
